src/app/utils/utils.ts

```diff
@@ -0,0 +1,28 @@
+export function addZeroToTime(i: int) {
+    if (i < 10) { return "0" + i }
+    return i;
+};
+
+export function getDateAsddMMyyyy(date: Date) {
+    return `${addZeroToTime(date.getDate())}/${date.getMonth() + 1}/${date.getFullYear()}`;
+};
+
+export function getDateAsDDTTTT(date: Date) {
+    return `${date.getDate()} / ${date.getHours()}:${addZeroToTime(date.getMinutes())}`
+};
+
+export function getDatesArray(startDate: Date, stopDate: Date) {
+    let dateArray = new Array();
+    let currentDate = startDate;
+    while (currentDate <= stopDate) {
+        dateArray.push(new Date(currentDate));
+        currentDate = addDays(currentDate, 1);
+    }
+    return dateArray;
+};
```

Comment:
```suggestion
export function getDatesArray(startDate: Date, stopDate: Date): Date[] {
    const dateArray: Date[] = [];
    let currentDate = startDate;
    while (currentDate <= stopDate) {
        dateArray.push(new Date(currentDate));
        currentDate = addDays(currentDate, 1);
    }
    return dateArray;
}
```

No need for a `let`, when you are not modifying variable reference. You are mutating existing variable, not creating something else instead of existing, so it is preferred to use `const` instead of `let`.

Also, instead of `new Array()` use `[]` if you do not pass anything to `Array` constructor, this way it is simpler and shorter.

Thing with `const` applies to a lot of the places in this repo

Reply:
Oh, and typing, don't forget about the typings